nginx.conf

```diff
@@ -37,7 +37,7 @@ http {
    add_header Pragma "no-cache";
 
    # Content Security Policy
-   add_header Content-Security-Policy "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com; script-src 'self' 'unsafe-eval' *.gov.bc.ca *.hotjar.com; style-src 'self' 'unsafe-inline' *.cloudflare.com; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com; img-src 'self' *.postescanada-canadapost.ca data:; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io; manifest-src 'self';";
+   add_header Content-Security-Policy "default-src 'self'; frame-src 'self' *.gov.bc.ca *.hotjar.com; script-src 'self' 'unsafe-eval' *.gov.bc.ca *.hotjar.com; style-src 'self' 'unsafe-inline' *.cloudflare.com; font-src 'self' *.gov.bc.ca *.hotjar.com *.cloudflare.com; img-src 'self' *.postescanada-canadapost.ca data:; connect-src 'self' *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io *.apigee.net; manifest-src 'self';";
```

Comment:
all 'src' sections need to add the domain.

Reply:
But this is now the same add_header as Create UI.

Does Create UI need to be updated too?

Reply:
Or maybe adding the extra src sections is out of scope for THIS problem?

Reply:
It seems you added the domain into image-src section of the config.

Reply:
Never mind. It is my issue. My eyes get blur.